File: src/Students.Domain/Enums/StudyFormat.cs
using System.ComponentModel.DataAnnotations;

namespace Students.Domain.Enums;

public enum StudyFormat
{
    [Display(Name = "Очно")]
    FullTime = 1,

    [Display(Name = "Заочно")]
    PartTime = 2,

    [Display(Name = "Вечернее")]
    Evening = 3
}

File: src/Students.BackOffice/client-app/src/domain/StudentGroups/StudyFormat.ts
export enum StudyFormat {
	FullTime = 1,
	PartTime = 2,
	Evening = 3
}

export namespace StudyFormat {
	export const getDisplayName = (category: StudyFormat): string => {
		switch (category) {
			case StudyFormat.FullTime:
				return 'Очная';
			case StudyFormat.PartTime:
				return 'Заочная';
			case StudyFormat.Evening:
				return 'Вечерняя';
			
		}
	};
}
